perfdash/perfdash.go

```diff
@@ -32,22 +32,27 @@ const (
 	maxBuilds    = 100
 )
 
-var (
-	addr        = pflag.String("address", ":8080", "The address to serve web data on")
-	www         = pflag.Bool("www", false, "If true, start a web-server to server performance data")
-	wwwDir      = pflag.String("dir", "www", "If non-empty, add a file server for this directory at the root of the web server")
-	builds      = pflag.Int("builds", maxBuilds, "Total builds number")
-	configPaths = pflag.StringArray("configPath", []string{}, "Paths/urls to the prow config")
+var options = &GoogleGCSDownloaderOptions{}
 
-	logsBucket     = pflag.String("logsBucket", "kubernetes-jenkins", "Name of the data bucket")
-	logsPath       = pflag.String("logsPath", "logs", "Path to the logs inside the logs bucket")
-	credentialPath = pflag.String("credentialPath", "", "Path to the gcs credential json")
+var (
+	addr   = pflag.String("address", ":8080", "The address to serve web data on")
+	www    = pflag.Bool("www", false, "If true, start a web-server to server performance data")
+	wwwDir = pflag.String("dir", "www", "If non-empty, add a file server for this directory at the root of the web server")
 
 	storageUrl = pflag.String("storageUrl", "https://k8s-gubernator.appspot.com/build", "Name of the data bucket")
 
 	globalConfig = make(map[string]string)
 )
 
+func initGoogleDownloaderOptions() {
+	pflag.BoolVar(&options.OverrideBuildCount, "override-build-count", false, "Whether to override number of build to process defined by \"perfDashBuildsCount\"")
```

Comment:
Sorry, but for someone who has no knowledge of perf-dash code (me), this is really hard to understand. 
Could you define what this flag/feature does without referring to existing perf-dash intricacies (i.e. perfDashBuildsCount - I have no idea what it is / where it is defines)?

Reply:
Changed flags' name and  description to be hopefully easier to understand by someone not familiar with PerfDash.